README.rst

```diff
@@ -1,17 +1,17 @@
 +---------------+-----------------+
 | Service       | Master branch   |
 +===============+=================+
-| Travis        | |travis_ci|     |
+| GitHub        | |ghactions_ci|  |
 +---------------+-----------------+
 | Appveyor      | |appveyor_ci|   |
 +---------------+-----------------+
 | Read the Docs | |read_the_docs| |
 +---------------+-----------------+
 
-.. |travis_ci| image:: https://api.travis-ci.org/PyWavelets/pywt.svg?branch=master
-   :align: middle
-   :target: https://travis-ci.org/PyWavelets/pywt
-   :alt: TravisCI Status
+
+.. |ghactions_ci| image:: https://github.com/PyWavelets/pywt/actions/workflows/tests/badge.svg?branch=master
+   :alt: Build Status
+   :target: https://github.com/PyWavelets/pywt/actions/workflows/tests.yml?query=branch%3Amaster
```

Comment:
Did you mean for this to be a text link to the actions page, or should this also be https://github.com/PyWavelets/pywt/actions/workflows/tests.yml/badge.svg?branch=master

Reply:
Fixed.